consts.go

```diff
@@ -0,0 +1,13 @@
+package centraldogma
+
+import (
+	"fmt"
+)
+
+var (
+	ErrLatestNotSet = fmt.Errorf("latest is not set yet")
```

Comment:
(`ErrLatestIsNotSet` and `ErrWatchedIsClosed`) or (`ErrLatestNotSet` and `ErrWatcherClosed`) for consistency.